### PR TITLE
Match func_8026EBF8, LoadFreeItemScript, and LoadItemScript

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Comment by JoshDuMan** [comment]
LoadItemScript - https://decomp.me/scratch/ERRgD
func_8026EBF8 - https://decomp.me/scratch/zGMkp

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.02%) on `us`.

⚠️ This PR introduces a warning:
- src/1967B0.c:60: warning: assignment makes integer from pointer without a cast
- src/1967B0.c:72: warning: passing arg 1 of `dma_copy' makes pointer from integer without a cast
- src/1967B0.c:72: warning: passing arg 2 of `dma_copy' makes pointer from integer without a cast
- src/1967B0.c:72: warning: passing arg 3 of `dma_copy' makes pointer from integer without a cast
- src/1967B0.c:78: warning: implicit declaration of function `enable_player_blur'
- src/197F40.c:2402: warning: implicit declaration of function `func_80266E40'
- src/197F40.c:2539: warning: implicit declaration of function `remove_player_buffs'
- src/197F40.c:2629: warning: initialization makes pointer from integer without a cast
- src/197F40.c:2633: warning: comparison between pointer and integer
- src/197F40.c:2634: warning: assignment makes pointer from integer without a cast
- src/197F40.c:2638: warning: passing arg 1 of `get_actor' makes integer from pointer without a cast
- src/197F40.c:2640: warning: implicit declaration of function `is_actor_hp_bar_visible'
- src/197F40.c:2641: warning: implicit declaration of function `load_tattle_flags'
- src/197F40.c:2656: warning: implicit declaration of function `func_80251474'

File: include/common_structs.h [hunk 1]
```diff
@@ -1447,7 +1447,15 @@ typedef struct DecorationTable {
     /* 0x6CD */ s8 unk_6CD;
     /* 0x6CE */ char unk_6CE[6];
     /* 0x6D4 */ s32* unk_6D4;
-    /* 0x6D8 */ char unk_6D8[120];
+    /* 0x6D8 */ char unk_6D8[0x67];
```
Comment (ethteck) [inline include/common_structs.h:1450]: this should be 0x68

File: src/1967B0.c [hunk 2]
```diff
@@ -13,6 +11,58 @@ typedef struct BattleItemTableEntry {
...
+        if (playerData->invItems[i] == itemID) {
+            playerData->invItems[i] = 0;
+            break;
+        }
+        i++;
+    }
+    sort_items();
+
+    if (battleStatus->selectedMoveID == MOVE_DOUBLE_DIP) {
+        if ((s8) battleStatus->itemUsesLeft == 2) {
```
Comment (ethteck) [inline src/1967B0.c:32]: Can we make this an s8 to avoid the cast?

**Review by ethteck — CHANGES_REQUESTED** [review]

**Review by ethteck — APPROVED** [review]

**Review by ethteck — APPROVED** [review]

File: src/1967B0.c [hunk 3]
```diff
@@ -13,7 +11,95 @@ typedef struct BattleItemTableEntry {
...
+
+    if (battleStatus->selectedMoveID == MOVE_TRIPLE_DIP) {
+        if (battleStatus->itemUsesLeft == 3) {
+            deduct_current_move_fp();
+        }
+    }
+    
+    itemPtr = &D_80293B80[0];
+    for (i = 0; *itemPtr != ITEM_NONE; i++, itemPtr++) {
+        if (*itemPtr == battleStatus->selectedItemID){
```
Comment (nanaian) [inline src/1967B0.c:45]:
```suggestion
        if (*itemPtr == battleStatus->selectedItemID) {
```

**Review by nanaian — CHANGES_REQUESTED** [review]

File: src/1967B0.c (hunk 3 above)
Comment (ethteck) [reply src/1967B0.c:45]: ohh nice, nabbed 'em

**Pull request merged**